addons/mrp/models/mrp_bom.py

```diff
@@ -409,9 +409,9 @@ def _get_default_product_uom_id(self):
             'You should install the mrp_byproduct module if you want to manage extra products on BoMs !'),
     ]
 
-    @api.depends('product_id', 'tracking')
+    @api.depends('product_id', 'tracking', 'operation_id')
     def _compute_manual_consumption(self):
-        self.filtered(lambda m: m.tracking != 'none').manual_consumption = True
+        self.filtered(lambda m: m.tracking != 'none' or (m.operation_id and m.operation_id.name)).manual_consumption = True
```

Comment:
Why checking if the operation has a name ? The operation itself should be enough (falsy if empty record field)
```suggestion
        self.filtered(lambda m: m.tracking != 'none' or m.operation_id).manual_consumption = True
```